core/src/main/java/nl/nn/adapterframework/batch/StreamTransformerPipe.java

```diff
@@ -68,6 +68,7 @@
 public class StreamTransformerPipe extends FixedForwardPipe {
 
 	public static final String originalBlockKey="originalBlock";
+	private final String deprecationSuppressKey="warnings.suppress.deprecated";
```

Comment:
Move this, as public final static, to ConfigurationWarnings so all warnings/keys are in 1 place.